integration/util_test.go

```diff
@@ -176,6 +178,7 @@ services:
    - REGISTRY_STORAGE_DELETE_ENABLED=true
   volumes:
    - {{.AuthDir}}:/auth:ro
+   - {{.AuthDir}}/config.json:/etc/zot/config.json
```

Comment:
Does this really have anything to do with the AuthDir?

Reply:
My reasoning for placing it here was that it helped point the zot image to the right auth files, and thus it somewhat made sense.

I could alternatively make another tempdir (mabye call it config?) and put the config files in there. That way if we ever need to use any config files in the future it could be placed there. But it would also be largely unused and I don't know how expensive an extra tempdir for every single shell ever created in our testing suite would be. What do you think?

Reply:
I think it might be good to change `AuthDir` to a generic `HostVolumeDir` that contains `auth/*` and `zot/config.json` so we can put arbitrary data that we want to mount from shell container -> registry container. 

If it’s really static across all shells, maybe we could bake it in to the integ test docker file.

Reply:
It's static across all shells but it's also unnecessary. I think we could probably redo this function a bit so that we aren't loading an extra zot config for every single time we get a shell with a registry. Currently it's probably fine but if our testing suite ends up encompassing more images that need more files, it might get a bit clunky.

Reply:
Ok, I made some changes to help make the separation of ideas more clear. Hopefully this is better?